packages/modeling/src/geometries/poly3/create.js

```diff
@@ -4,6 +4,8 @@
  * instances but they must behave similarly.
  * @typedef {Object} poly3
  * @property {Array} vertices - list of ordered vertices (3D)
+ * @example
+ * {"vertices": [[0,0,0], [4,0,0], [4,3,12]]}
  */
```

Comment:
Not specific to this PR, but shouldn't this whole comment be in `index.js`?

Reply:
definitely agree. it would be easier to find information about the data structures if explained in the index, not the create function.

fyi, something happened with the jsdoc/docdash templates. there are no longer 'parameters'. i have tried several different fixes but still not able to determine the reason. i'm not concerned but this has to be fixed.